src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -2328,13 +2328,48 @@ bool os::Bsd::hugetlbfs_sanity_check(bool warn, size_t page_size) {
 
 // Large page support
 
-static size_t _large_page_size = 0;
+static size_t _large_page_size =
```

Comment:
This value is exposed to shared hotspot code via os::large_page_size(). Please use `#if defined(__FreeBSD__)` for the non-zero values.

Reply:
Ah correct, OpenBSD has no equivalent feature, not sure about NetBSD but do not think neither. MacOS would possibly be the only "bsd" which would share few values of these.